appengine/standard/django/requirements.txt

```diff
@@ -1,2 +1,2 @@
 PyMySQL==0.9.3
-Django<2.0.0,>=1.11.8
```

Comment:
I know we have Django as a range like this for a reason - I believe 2.0.0 won't work with Python 2.7. Should we pin it directly to 1.11.27 but add a comment to the requirements.txt that it needs to stay under 2.0.0?

Reply:
Ah I didn't think about that - we should make sure to scope out which samples are still supporting Python2 and create rules in Renovate for those.